news/2018-11-30-new-featured-artist-soooo.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: SOOOO"
+date: 2018-11-30 20:00:00 +0000
+---
+
+The Featured Artist project storms into the sounds of a turbulent youth with the latest addition to the roster, **SOOOO!**
```

Comment:
to its roster

Reply:
i'm kind of hesitant to change this, 'to its roster' feels wrong/worse here?

anyone else weigh in?

Reply:
"its" makes more sense cuz "featured artist project" is the subject of sentence

Reply:
teams don't really say "add to its roster" though, they say "add to THE roster"

which is sort of what i'm gunning for